arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,440 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/replicationLinks/{linkId}": {
+            "delete": {
+                "tags": [
+                    "DatabaseReplicationLinks"
+                ],
```

Comment:
A replication link represents a GeoDR link between two databases.  I would prefer to keep them nested under databases rather than elevate them to being their own unique operation group.

---
In reply to: [80162927](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80162927) [](ancestors = 80162927)